generators/python-v2/codegen/src/ast/Operator.ts

```diff
@@ -0,0 +1,49 @@
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+import { assertNever } from "@fern-api/core-utils";
+
+type OperatorType = "or" | "and" | "bitwiseLeftShift" | "bitwiseRightShift";
```

Comment:
We at Vellum actually only _really_ need `bitwiseRightShift` currently.

Reply:
Thoughts on defining an enum (like [Access](https://github.com/fern-api/fern/blob/9be2af2973a32c913af251e35c84cb7626231808/generators/php/codegen/src/ast/Access.ts#L3)) for this so it's a little easier to read at the call-site?

We could also remove `bitwise` since we're unlikely to need to use `leftShift` or `rightShift` for any other reason. It doesn't need to be an AstNode instance, but it could still be defined in its own `OperatorType.ts` file for discoverability:

```typescript
// OperatorType.ts
export type OperatorType = "or" | "and" | "leftShift" | "rightShift";

export const OperatorType = {
    Or: "or",
    And: "and",
    LeftShift: "leftShift",
    RightShift: "rightShift"
} as const;
```

From your test, we could instead write:
```typescript
const operator = python.operator({
    operator: OperatorType.LeftShift,
    lhs: python.TypeInstantiation.int(1),
    rhs: python.TypeInstantiation.int(2)
});
```

Reply:
Yep sounds good!